BoxContentSDK/BoxContentSDK/Operations/BOXAPIMultipartToJSONOperation.m

```diff
@@ -169,9 +191,8 @@ - (NSDictionary *)HTTPHeaders
 
 - (void)abortWithError:(NSError *)error
 {
-    [self.connection cancel];
+    self.error = error;
```

Comment:
Is there not a callback we need to invoke to propagate up the error or is assigning it to self.error sufficient?